src/hooks/useFormatNumber.mdx

```diff
@@ -0,0 +1,49 @@
+import { Meta, Source } from "@storybook/blocks";
+
+<Meta title="Hooks/useFormatNumber" />
+
+# useFormatNumber
+
+| Hook Version | DS Version   |
+| ------------ | ------------ |
+| Added        | `Prerelease` |
+| Latest       | `Prerelease` |
+
+`useFormatNumber` provides a number with commas and handles a range of numbers.
+The formatting is based on standards from the NYPL Writing Style Guide.
+
+## Usage
+
+const formatNumber from "../../hooks/useFormatNumber";
```

Comment:
This code snippet should show the code that consuming apps can use to import the hook/function.
```
<Source
  code={`
import {
  formatNumber
} from "@nypl/design-system-react-components";
`} language="jsx" />
```